extensions/resteasy-reactive/rest-client-reactive/deployment/src/main/java/io/quarkus/rest/client/reactive/deployment/ReactiveResteasyMpClientProcessor.java

```diff
@@ -146,6 +154,58 @@ void registerHeaderFactoryBeans(CombinedIndexBuildItem index,
         }
     }
 
+    @BuildStep
+    void registerProviders(CombinedIndexBuildItem combinedIndexBuildItem,
```

Comment:
This was copied almost verbatim from the old rest-client

Reply:
this doesn't seem to be necessary. Registering client headers and providers is now handled by code generated in the build time.
Maybe some specific part of this method is needed?

Reply:
I'll try, but I am pretty sure it wasn't without this

Reply:
This indeed is no needed